cli/src/commit_templater.rs

```diff
@@ -479,6 +510,8 @@ pub enum CommitTemplatePropertyKind<'repo> {
         Box<dyn TemplateProperty<Output = Option<CryptographicSignature>> + 'repo>,
     ),
     AnnotationLine(Box<dyn TemplateProperty<Output = AnnotationLine> + 'repo>),
+    Trailer(Box<dyn TemplateProperty<Output = Trailer> + 'repo>),
+    Trailers(Box<dyn TemplateProperty<Output = Vec<Trailer>> + 'repo>),
```

Comment:
nit: `TrailerList` (and `trailer_list`) per convention

Reply:
should I also change the exposed method name?

Reply:
> should I also change the exposed method name?

You mean `commit.trailers()` to `.trailer_list()`? No. It's not a function/enum that processes an object of certain template type.